doc/rst/source/datasets/earth-relief.rst

```diff
@@ -22,36 +22,40 @@ We have processed and reformatted publicly available global relief
-Code Dimensions        Size     Description
-==== ================= =======  ==================================================
-01d       361 x    181  106 KB  1 arc degree global relief (SRTM15+V2.1 @ 111 km)
-30m       721 x    361  363 KB  30 arc minute global relief (SRTM15+V2.1 @ 55 km)
-20m      1081 x    541  759 KB  20 arc minute global relief (SRTM15+V2.1 @ 37 km)
-15m      1441 x    721  1.3 MB  15 arc minute global relief (SRTM15+V2.1 @ 28 km)
-10m      2161 x   1081  2.8 MB  10 arc minute global relief (SRTM15+V2.1 @ 18 km)
-06m      3601 x   1801  7.3 MB  6 arc minute global relief (SRTM15+V2.1 @ 10 km)
-05m      4321 x   2161   10 MB  5 arc minute global relief (SRTM15+V2.1 @ 9 km)
-04m      5401 x   2701   16 MB  4 arc minute global relief (SRTM15+V2.1 @ 7.5 km)
-03m      7201 x   3601   27 MB  3 arc minute global relief (SRTM15+V2.1 @ 5.6 km)
-02m     10801 x   5401   58 MB  2 arc minute global relief (SRTM15+V2.1 @ 3.7 km)
-01m     21601 x  10801  215 MB  1 arc minute global relief (SRTM15+V2.1 @ 1.9 km)
-30s     43201 x  21601  769 MB  30 arc second global relief (SRTM15+V2.1 @ 1.0 km)
-15s     86400 x  43200  2.6 GB  15 arc second global relief (SRTM15+V2.1)
-03s    432001 x 216001  6.8 GB  3 arc second global relief (SRTM3S)
-01s   1296001 x 432001   41 GB  1 arc second global relief (SRTM1S)
-==== ================= =======  ==================================================
+==== ================= === =======  ==================================================
+Code Dimensions        Reg Size     Description
+==== ================= === =======  ==================================================
+01d       361 x    181 g,p  128 KB  1 arc degree global relief (SRTM15+V2.1 @ 111 km)
+30m       721 x    361 g,p  435 KB  30 arc minute global relief (SRTM15+V2.1 @ 55 km)
+20m      1081 x    541 g,p  918 KB  20 arc minute global relief (SRTM15+V2.1 @ 37 km)
+15m      1441 x    721 g,p  1.6 MB  15 arc minute global relief (SRTM15+V2.1 @ 28 km)
+10m      2161 x   1081 g,p  3.4 MB  10 arc minute global relief (SRTM15+V2.1 @ 18 km)
+06m      3601 x   1801 g,p  8.8 MB  6 arc minute global relief (SRTM15+V2.1 @ 10 km)
+05m      4321 x   2161 g,p   13 MB  5 arc minute global relief (SRTM15+V2.1 @ 9 km)
+04m      5401 x   2701 g,p   19 MB  4 arc minute global relief (SRTM15+V2.1 @ 7.5 km)
+03m      7201 x   3601 g,p   33 MB  3 arc minute global relief (SRTM15+V2.1 @ 5.6 km)
+02m     10801 x   5401 g,p   71 MB  2 arc minute global relief (SRTM15+V2.1 @ 3.7 km)
+01m     21601 x  10801 g,p  258 MB  1 arc minute global relief (SRTM15+V2.1 @ 1.9 km)
+30s     43201 x  21601 g,p  935 MB  30 arc second global relief (SRTM15+V2.1 @ 1.0 km)
+15s     86400 x  43200 p    3.2 GB  15 arc second global relief (SRTM15+V2.1)
+03s    432001 x 216001 p    6.8 GB  3 arc second global relief (SRTM3S)
+01s   1296001 x 432001 p     41 GB  1 arc second global relief (SRTM1S)
+==== ================= === =======  ==================================================
```

Comment:
There is a similar table in `doc/rst/source/earth_relief.rst_`. We also need to update that table.

Reply:
yes, I need to update all of that stuff with new informaiton.